src/Integration.Vsix/SonarLintNotificationsPackage.cs

```diff
@@ -76,21 +76,23 @@ private async System.Threading.Tasks.Task InitAsync()
 
             notifications = new SonarQubeNotificationService(sonarqubeService,
                 new NotificationIndicatorViewModel(), new TimerWrapper { Interval = 60000 }, logger);
-
-            activeSolutionBoundTracker = await this.GetMefServiceAsync<IActiveSolutionBoundTracker>();
-
-            // A bound solution might already have completed loading. If so, we need to
-            // trigger the load of the options from the solution file
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode != SonarLintMode.Standalone)
-            {
-                var solutionPersistence = (IVsSolutionPersistence)await this.GetServiceAsync(typeof(SVsSolutionPersistence));
-                solutionPersistence.LoadPackageUserOpts(this, NotificationDataKey);
-            }
-
```

Comment:
This section hasn't changed - I just moved it inside the the block below that is executed on the UI thread.